### PR TITLE
Preneeds: convert `nationalGuardState` to enum

### PR DESCRIPTION
**Pull request by jholton**

### CODE REVIEW

**Comment by jholton** [comment]
Yes, I asked the email thread we have going with PreNeeds stakeholders for confirmation of the states.  I'll update the list when I get them.

**Comment by cehsu** [comment]
Thanks for the context.

File: src/common/constants.js [hunk 1]
```diff
@@ -4207,33 +4210,33 @@ const pciuStates = states.USA
...
-    { value: 'L107', label: 'VA Form 21-4142 - Authorization To Disclose Information' },
-    { value: 'L827', label: 'VA Form 21-4142a - General Release for Medical Provider Information' },
-    { value: 'L115', label: 'VA Form 21-4192 - Request for Employment Information in Connection with Claim for Disability' },
-    { value: 'L117', label: 'VA Form 21-4502 - Application for Automobile or Other Conveyance and Adaptive Equipment Under 38 U.S.C. 3901-3904' },
-    { value: 'L159', label: 'VA Form 26-4555 - Application in Acquiring Specially Adapted Housing or Special Home Adaptation Grant' },
-    { value: 'L133', label: 'VA Form 21-674 - Request for Approval of School Attendance' },
-    { value: 'L139', label: 'VA Form 21-686c - Declaration of Status of Dependents' },
-    { value: 'L149', label: 'VA Form 21-8940 - Veterans Application for Increased Compensation Based on Un-employability' }
-  ];
+const documentTypes526 = [
```
Comment (jholton) [inline src/common/constants.js]: just fixing indentation.

File: src/schemas/40-10007/schema.js [hunk 2]
```diff
@@ -87,7 +94,10 @@ _.merge(modifiedToursOfDuty, {
         type: 'string'
       },
       nationalGuardState: {
-        type: 'string'
+        type: 'string',
+        maxLength: 3,
+        'enum': nationalGuardStates.map(state => state.value),
+        enumNames: nationalGuardStates.map(state => state.label)
```
Comment (jholton) [inline src/schemas/40-10007/schema.js]: Is this useful for the frontend?  @cehsu @erikphansen

Comment (jholton) [reply src/schemas/40-10007/schema.js]: That's what I thought.  Thanks.

**Pull request merged**